de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/grid/GridViewPresenterImpl.java

```diff
@@ -276,6 +276,11 @@ public void onDiskResourceSelectionChanged(DiskResourceSelectionChangedEvent eve
             // Only call get stat for single selections
             return;
         }
+
+        if(selection.size() > 0 && selection.iterator().next().isFilter()) {
```

Comment:
Prior to this, we have a check to make sure the selection size is 1 or else we return, so we can probably simplify this.